Install guide review (#677)
SUSE/doc-susemanager: modules/installation/pages/general-requirements.adoc

[[installation-general-requirements]]
= General Requirements

// REMARK this list needs to be cleaned up

> Before you begin installation, ensure that your environment meets the following requirements:

Reviewer: ```suggestion
Before you begin installation, ensure that your environment meets these requirements:
```

Directional language.

[[install.scc-register]]
== Obtaining your SUSE Customer Center Credentials

> Create an account with {scc} before installation of {sls} and {productname}.

Reviewer: ```suggestion
Create an account with {scc} before installation of {sls} and {productname}.
```

Why did you make this passive? The original worked better.

Author: Woops must have been a little tired thanks!

[[creating.scc.account.mgr]]
.Procedure: Obtaining Your SCC Organization Credentials
. Navigate to https://scc.suse.com/login in your Web browser.

> . Log in to your SCC account, or follow the prompts to create a new account.

Reviewer 2: In the following, these are not buttons but other GUI elements.  Thus I'd rather vote to use [guimenu] (or menu: if appropriate).
